web/docs/tql2/operators/context/create_geoip.md

```diff
@@ -0,0 +1,54 @@
+# create_geoip
+
+Creates a GeoIP context.
+
+```tql
+context::create_geoip name:string [db_path:string]
+```
+
+## Description
+
+The `context::create_geoip` operator constructs a new context of type
+[GeoIP](../../../enrichment/README.md#geoip).
+
+You must either provide a database with the `db_path` argument or use
+[`context::load`](load.md) to populate the context after creation.
+
+### `name : string`
+
+The name of the new GeoIP context.
+
+### `db_path : string (optional)`
+
+The path to the [MMDB](https://maxmind.github.io/MaxMind-DB/) database.
```

Comment:
This should probably explain that paths are relative to the node's working directory.